libraries/streaming-e2e-tests/react-app/package.json

```diff
@@ -6,14 +6,15 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "botframework-webchat": "4.9.2",
+    "botframework-webchat": "^4.9.2",
```

Comment:
Currently it uses at least version 4.9.2 of WC
Can pin it to the latest patch version either by merging this in later or just in a separate small PR--depends on ordering of William's PR, I guess

Reply:
In the meanwhile, please grab the tarballs from https://github.com/microsoft/BotFramework-WebChat/releases/daily/ and install them in the following order:

```sh
npm i botframework-webchat-core-4.10.1-master.xxx.tgz
npm i botframework-webchat-component-4.10.1-master.xxx.tgz
npm i botframework-directlinespeech-sdk-4.10.1-master.xxx.tgz
npm i botframework-webchat-4.10.1-master.xxx.tgz
```

Reply:
Update: we'll merge this test pipeline after `botframework-webchat 4.10.1` is on npm, since getting the TGZ's will make the pipeline temporarily more complex that we would need to test to make sure that the steps play nicely in Azure Pipelines environment--and this extra work put in would be for a short-lived state (i.e. only about 2 days until bf-webchat is on npm) before we'd change it

package.json will be updated`botframework-webchat: 4.10.1`
* This will also affect the package-lock.json currently in PR and would need to be updated as well

Reply:
package.json and package-lock updated to now-released botframework-webchat version 4.10.1 ✔️